language/move-lang/tests/move_check/typing/missing_acquire.exp

```diff
@@ -0,0 +1,44 @@
+error: 
+
+    ┌── tests/move_check/typing/missing_acquire.move:8:9 ───
+    │
+ 8 │         r1(a);
+    │         ^^^^^ Invalid call to '0x8675309.M.r1'
+    ·
+ 14 │     r1(a: address) acquires R1 {
+    │                             -- The call acquires '0x8675309.M.R1', but the 'acquires' list for the current function does not contain this type. It must be present to make this call
```

Comment:
Nit: The phrase  "must be present to make this call" is pretty ambiguous. Maybe rephrase to something like "must be present in the calling context acquires list" or something along those lines?